packs/softlayer/actions/create_instance.yaml

```diff
@@ -0,0 +1,47 @@
+description: 'Creates a new instance'
+enabled: true
+entry_point: create_instance.py
+name: create_instance
+parameters:
+  name:
+    required: true
+    type: string
+    description: Hostname for the instance.
+  domain:
+    required: false
+    type: string
+    description: domain to attach to the instance (defaults to example.com).
+  cpus:
+    required: false
+    type: integer
+    description: Cpus for this instance (defaults to 1).
```

Comment:
Good practice it to also set the same default value in the action metadata file for all the parameters with default values.

This way this default value is also displayed when running `st2 action -h`.